sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestConsumer.java

```diff
@@ -35,27 +40,57 @@ public class TestConsumer {
     @RegisterExtension
     public JdbiExtension h2Extension = JdbiExtension.h2().withInitializer(TestingInitializers.something()).withPlugin(new SqlObjectPlugin());
 
-    @Test
-    public void testReturnStream() {
-        Something one = new Something(3, "foo");
-        Something two = new Something(4, "bar");
-        Something thr = new Something(5, "baz");
+    Spiffy dao;
+    List<Something> expected;
+
+    @BeforeEach
+    public void setup() {
+        final Something one = new Something(3, "foo");
+        final Something two = new Something(4, "bar");
+        final Something thr = new Something(5, "baz");
 
-        Spiffy dao = h2Extension.getSharedHandle().attach(Spiffy.class);
+        dao = h2Extension.getSharedHandle().attach(Spiffy.class);
         dao.insert(one);
         dao.insert(thr);
         dao.insert(two);
 
-        List<Something> results = new ArrayList<>();
+        expected = Arrays.asList(thr, two, one);
+    }
+
+    @Test
+    public void consumeEach() {
```

Comment:
btw, JUnit 5 can deal with package private test methods.

Reply:
so... if the extra "public" keyword does not bring any benefit... remove it? ;-) (SCNR)